### ISSUE TITLE
Game rules

### ISSUE THREAD

**Issue by stillonearth** [Ria8651/dicewars#1]
I was wondering where initial number of dice on each territory comes from. In your code you have `rng.gen_range(1..7)`.

**Comment by ria8651** [comment]
I just pick a random number between 1 and 7, and I just pick a random player for who gets each territory. This is very unfair as sometimes you get no territories or like 1 dice but works for now.

**Comment by stillonearth** [comment]
I guess this game is heavily based on https://en.wikipedia.org/wiki/Risk_(game)

`At the beginning of a player's turn, they receive reinforcement armies proportional to the number of territories held, bonus armies for holding whole continents, and additional armies for turning in matched sets of territory cards obtained by conquering new territories. The player may then attack, move their armies, or pass.`

This means that each player get equal amount of dice and then allocate them over the regions they control.

**Comment by ria8651** [comment]
right, that's what I do at the end of each turn, I randomly distribute 1 dice to your territories for each territory you own. I should just do that at the beginning too.
